packages/cli/generators/model/templates/model.ts.ejs

```diff
@@ -18,6 +22,15 @@ export class <%= className %> extends <%= modelBaseClass %> {
   <%= key %><%if (!val.required) {%>?<% } %>: <%= val.tsType %>;
 
 <% }) -%>
+
```

Comment:
I think this will create an empty line in the generated output, even when `allowAdditionalProperties` was set to `false`. Please move this empty line inside `<% if(allowAdditionalProperties) { -%>` branch.